<launch>
  <arg name="requestTopic"       default="true"/>
  <arg name="addressToSave"       default="/home/buivn/bui_ws/src/pahap/pointcloud/"/>



  <node name="pahap_client" pkg="pahap" type="pcl_pahap_client">
    <param name="requestTopic"   value="$(arg requestTopic)"/>
    <param name="addressToSave"  value="$(arg addressToSave)"/>
  </node>

</launch>
